CS483/CS483/Kartaclysm/StateMachine/Gameplay/StateRacing.cpp

```diff
@@ -90,6 +90,8 @@ void Kartaclysm::StateRacing::Enter(const std::map<std::string, std::string>& p_
 	// Store passed context parameters and begin race
 	m_mContextParams = p_mContextParameters;
 	BeginRace();
+
+	HeatStroke::AudioPlayer::Instance()->PlaySoundEffect("Assets/Sounds/engine.wav", true);
```

Comment:
Good for now, but we should probably have this tied to individual karts.  Would give more sound for more racers, and allow karts to have different sounds.